src/ecosystem/Ecosystem.jl

```diff
@@ -398,15 +405,36 @@ function coral_spec()::NamedTuple
 
     # Bleaching sensitivity of each coral group
     # Bozec et al., (2022)
-    bleaching_sensitivity = Float64[
+    bleaching_sensitivity = Array{Float64,2}(Float64[
         1.50 1.50 1.50 1.50 1.50 1.50  # Tabular Acropora Enhanced (Arborescent staghorn corals)
         1.50 1.50 1.50 1.50 1.50 1.50  # Tabular Acropora Unenhanced
         1.40 1.40 1.40 1.40 1.40 1.40  # Corymbose Acropora Enhanced
         1.40 1.40 1.40 1.40 1.40 1.40  # Corymbose Acropora Unenhanced
         0.25 0.25 0.25 0.25 0.25 0.25  # Small massives and encrusting
-        0.25 0.25 0.25 0.25 0.25 0.25] # Large massives
+        0.25 0.25 0.25 0.25 0.25 0.25]) # Large massives
     params.bleaching_sensitivity = bleaching_sensitivity'[:]
 
+    # Natural adaptation / heritability
```

Comment:
Are these derived from Figure 1 in Bairos-Novak?

Reply:
Nope. There's missing detail here, have updated the comments.